pom.xml

```diff
@@ -1267,7 +1267,7 @@
         <carbon.apimgt.ui.version>9.0.311</carbon.apimgt.ui.version>
 
         <!-- APIM Component Version -->
-        <carbon.apimgt.version>9.22.3</carbon.apimgt.version>
+        <carbon.apimgt.version>9.22.5-SNAPSHOT</carbon.apimgt.version>
```

Comment:
We do not upgrade to snapshot versions. Just upgrade to 9.22.5. 
The artifacts are already released in 9.22.5.

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455